examples/example-gauth/README.md

```diff
@@ -44,9 +44,15 @@ gcloud pubsub topics create Topic1
 [service account](https://cloud.google.com/docs/authentication/#service_accounts) in order to access
 Pub/Sub via gRPC APIs as described [here](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
```

Comment:
Move the note to here after "as described here" with the below.

Note: This step is unnecessary on Google platforms (Google App Engine / Google Cloud Shell / Google Compute Engine) as it will automatically use the in-built Google credentials.

Reply:
Fixed